src/models/poi/index.ts

```diff
@@ -1,12 +1,12 @@
-import { poiStatus } from "../../constants/model/poi";
+// import { poiStatus } from "../../constants/model/poi";
 
 interface Poi {
   id: string;
   data: PoiData;
   media: PoiMedia;
 }
 
-export type PoiStatus = keyof typeof poiStatus;
+// export type PoiStatus = keyof typeof poiStatus;
```

Comment:
Please delete the code segment instead of commenting it.
Also, please delete the related part.

For the code here, you may need to delete the following parts
- any other references to `PoiStatus` and related logics
- the whole file of `src/constants/model/poi/index.ts`
- any reference to `poiStatus` and `poiStatusMessageKeys` and related logics